src/test/java/com/gotocompany/firehose/evaluator/GrpcResponseCelPayloadEvaluatorTest.java

```diff
@@ -0,0 +1,127 @@
+package com.gotocompany.firehose.evaluator;
+
+import com.google.protobuf.Message;
+import com.gotocompany.firehose.consumer.GenericError;
+import com.gotocompany.firehose.consumer.GenericResponse;
+import com.gotocompany.firehose.consumer.TestMessage;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+public class GrpcResponseCelPayloadEvaluatorTest {
+
+    private static final String CEL_EXPRESSION = "GenericResponse.success == false && GenericResponse.errors.exists(e, e.code == \"400\")";
+    private PayloadEvaluator<Message> grpcPayloadEvaluator;
+
+    @Before
+    public void setup() {
+        this.grpcPayloadEvaluator = new GrpcResponseCelPayloadEvaluator(GenericResponse.getDescriptor(), CEL_EXPRESSION);
```

Comment:
Nitpick; 

Since this `grpcPayloadEvaluator` is used in couple of test cases and for the rest of them we are creating it inline and due to java/junit scoped variable value isn't available, we can have helper method which could be used for all test cases.

For example:
@Test
    public void shouldEvaluateResponseToTrueWhenCELExpressionMatchesPayload() {
        GrpcPayloadEvaluator grpcPayloadEvaluator = forCELExpression(".......");
       .....
}

....



private static GrpcPayloadEvaluator forCELExpression(String ...) {
       return new ....
}

This is most likely DSL but improves test code readability imo.

Reply:
Yes, thanks for pointing out, I'll create a utils method for that case